lib/cc/engine/violation_decorator.rb

```diff
@@ -0,0 +1,41 @@
+module CC
+  module Engine
+    class ViolationDecorator < SimpleDelegator
+      MULTIPLIER_REGEX = %r{\[([\d\.]+)\/([\d\.]+)\]}
+
+      DEFAULT_REMEDIATION_POINTS = 200_000
+
+      def remediation_points
+        points = cop_list.fetch(cop_name, DEFAULT_REMEDIATION_POINTS)
+        if multiplier?
+          points * multiply_by
+        else
+          points
+        end
+      end
+
+      private
+
+      def multiply_by
+        result = message.scan(MULTIPLIER_REGEX)
+        result[0][0].to_i
+      end
+
+      def multiplier?
+        message.match(MULTIPLIER_REGEX)
+      end
+
+      def cop_list
+        return @cop_list if @cop_list
+
+        cops_path = File.expand_path(
+          File.join(File.dirname(__FILE__), "../../../config/cops.yml")
+        )
+
+        File.open(cops_path, "r") do |file|
+          @cop_list = YAML.load(file.read)
```

Comment:
Can we use `SafeYAML` here? Even though it's not user input, better to just always use that everywhere I think. 

Also, I think there's a `.load_file` method (on both modules).